### PR TITLE
Feature/new UI

### PR DESCRIPTION
**Pull request by brianshattuck**

### CODE REVIEW

File: src/config/layerx.json [hunk 1]
```diff
@@ -65,7 +70,7 @@
     "available": false
   },
   "hydra": {
-    "available": false
+    "available": true
```
Comment (sameepsi) [inline src/config/layerx.json:73]: @brianshattuck why this is changed from false to true?

File: src/config/layerx.json [hunk 2]
```diff
@@ -86,7 +91,7 @@
     "available": false
   },
   "steer": {
-    "available": true
+    "available": false
```
Comment (sameepsi) [inline src/config/layerx.json:94]: @brianshattuck why this has been done?

File: src/constants/v3/addresses.ts [hunk 3]
```diff
@@ -1732,3 +1775,34 @@ export class ExtendedEther extends V3Currency {
...
+  3: '0xc778417E063141139Fce010982780140Aa0cD5Ab', // Ropsten Testnet (WETH)
+  4: '0xc778417E063141139Fce010982780140Aa0cD5Ab', // Rinkeby Testnet (WETH)
+  5: '0xc778417E063141139Fce010982780140Aa0cD5Ab', // Goerli Testnet (WETH)
+  42: '0xd0A1E359811322d97991E03f863a0C30C2cF029C', // Kovan Testnet (WETH)
+  56: '0xBB4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c', // Binance Smart Chain Mainnet (WBNB)
+  97: '0xae13d989dac2f0debff460ac112a837c89baa7cd', // Binance Smart Chain Testnet (WBNB)
+  137: '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270', // Polygon Mainnet (WMATIC)
+  80001: '0x9c3c9283d3e44854697cd22d3faa240cfb032889', // Mumbai Testnet (WMATIC)
+  43114: '0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7', // Avalanche Mainnet (WAVAX)
+  1101: '0xa2036f0538221a77A3937F1379699f44945018d0', // Polygon zkEVM Mainnet (WETH)
```
Comment (sameepsi) [inline src/constants/v3/addresses.ts:1790]: @brianshattuck this is not WETH but MATIC on zkevm

File: src/constants/v3/addresses.ts [hunk 4]
```diff
@@ -1732,3 +1775,34 @@ export class ExtendedEther extends V3Currency {
...
+  344: '0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7', // Manta Pacific L2 Rollup (WETH)
```
Comment (sameepsi) [inline src/constants/v3/addresses.ts:1791]: @brianshattuck this is not weth on manta

File: src/constants/v3/addresses.ts [hunk 5]
```diff
@@ -1732,3 +1775,34 @@ export class ExtendedEther extends V3Currency {
...
+  132: '0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7', // Immutable zkEVM Mainnet (WETH)
```
Comment (sameepsi) [inline src/constants/v3/addresses.ts:1792]: This is wrong as well

File: src/constants/v3/addresses.ts [hunk 6]
```diff
@@ -1732,3 +1775,34 @@ export class ExtendedEther extends V3Currency {
...
+  592: '0x9fB83c0635De2E815fd1c21b3a292277540C2e8d', // Astar zkEVM (WASTR)
```
Comment (sameepsi) [inline src/constants/v3/addresses.ts:1793]: This is wrong as well

File: src/constants/v3/addresses.ts [hunk 7]
```diff
@@ -1732,3 +1775,34 @@ export class ExtendedEther extends V3Currency {
...
+  2000: '0x5c21a9226A2E5cd2c30bDB3A5D53E1c2325Ff55a', // DogeChain (wDOGE)
```
Comment (sameepsi) [inline src/constants/v3/addresses.ts:1794]: This is wrong as well

File: src/constants/v3/addresses.ts [hunk 8]
```diff
@@ -1732,3 +1775,34 @@ export class ExtendedEther extends V3Currency {
...
+  2010: '0xB31f66AA3C1e785363F0875A1B74E27b85FD66c7', // X Layer (WETH)
```
Comment (sameepsi) [inline src/constants/v3/addresses.ts:1795]: i am assuming this is wrong as well

File: src/constants/v3/addresses.ts [hunk 9]
```diff
@@ -1732,3 +1775,34 @@ export class ExtendedEther extends V3Currency {
...
+  2222: '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee', // Kava - Kinetix (WKAVA)
```
Comment (sameepsi) [inline src/constants/v3/addresses.ts:1796]: this is wrong as well

File: src/config/layerx.json (hunk 1 above)
Comment (Kalesberg) [reply src/config/layerx.json]: my branch was behind dev2 branch for a while, and that's why

File: src/config/layerx.json (hunk 2 above)
Comment (Kalesberg) [reply src/config/layerx.json]: same as above

File: src/constants/v3/addresses.ts (hunk 3 above)
Comment (Kalesberg) [reply src/constants/v3/addresses.ts]: fixed this

File: src/constants/v3/addresses.ts (hunk 4 above)
Comment (Kalesberg) [reply src/constants/v3/addresses.ts]: fixed this

File: src/constants/v3/addresses.ts (hunk 5 above)
Comment (Kalesberg) [reply src/constants/v3/addresses.ts]: fixed

File: src/constants/v3/addresses.ts (hunk 6 above)
Comment (Kalesberg) [reply src/constants/v3/addresses.ts]: fixed

File: src/constants/v3/addresses.ts (hunk 7 above)
Comment (Kalesberg) [reply src/constants/v3/addresses.ts]: fixed

File: src/constants/v3/addresses.ts (hunk 8 above)
Comment (Kalesberg) [reply src/constants/v3/addresses.ts]: fixed

File: src/constants/v3/addresses.ts (hunk 9 above)
Comment (Kalesberg) [reply src/constants/v3/addresses.ts]: fixed

**Review by sameepsi — APPROVED** [review]

**Review by sameepsi — APPROVED** [review]

**Pull request merged**